src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -167,6 +168,36 @@ public void cleanupDoiReturnsChangeWhenDoiInURLField() {
     }
 
     @Test
+    public void cleanupISSNReturnsCorrectISSN() {
```

Comment:
I think the reason is all the dependency injections. Now it wasn't that hard, just mock both arguments, but it is clearly quicker to just add them to existing tests...